test/e2e/kuma_deploy_multi_apps_test.go

```diff
@@ -65,14 +65,16 @@ metadata:
 
 		clientPod := pods[0]
 
-		_, stderr, err := c1.ExecWithRetries(TestNamespace, clientPod.GetName(), "demo-client",
-			"curl", "-v", "-m", "3", "echo-server")
-		Expect(err).ToNot(HaveOccurred())
-		Expect(stderr).To(ContainSubstring("HTTP/1.1 200 OK"))
+		Eventually(func() (string, error) {
+			_, stderr, err := c1.ExecWithRetries(TestNamespace, clientPod.GetName(), "demo-client",
+				"curl", "-v", "-m", "3", "echo-server")
+			return stderr, err
+		}, "10s", "1s").Should(ContainSubstring("HTTP/1.1 200 OK"))
```

Comment:
shall we use the default for the timeouts/

Reply:
Can we retry if not 200 OK returned?

Reply:
`Eventually` + `ExecWithRetries`

Can we have single code for retries?

Reply:
> Can we retry if not 200 OK returned?

@nickolaev I'm sorry, not sure that I'm following here. It will retry until a response contains `HTTP/1.1 200 OK`

Reply:
@jakubdyszkiewicz I believe `ExecWithRetries` should be named just `Exec`. These retries are needed to exec command with status code equals 0. And `eventually` is needed to check that response itself is correct

Reply:
Oh actually it is OK. Please ignore.

Reply:
I am tempted to have "CurlWithRetries" :)